### DISCUSSION TITLE
Can denoising be applied in a particle-wise situation?

### DISCUSSION THREAD

**Discussion by zhaotf16** [tbepler/topaz#139] [Q&A] [answered]
Thanks for your contribution. The denoising algorithm has been proved in many examples and has shown good performance. I have a small question.

Say, if we have a lot of particles instead of micrographs, (like several mrcs files ). Will this still work well?

Thanks!

**Comment by tbepler** [comment, marked as answer]
Topaz denoise can be run on particle stacks and seems to work fine, but, in my experience, it generally works better to denoise the whole micrographs and then get the particle stacks from those if you want denoised particles.
